unlock-app/src/components/interface/locks/Manage/elements/MetadataCard.tsx

```diff
@@ -0,0 +1,377 @@
+import { Button, Badge, Input, Modal } from '@unlock-protocol/ui'
+import { useState } from 'react'
+import { FieldValues, useForm } from 'react-hook-form'
+import {
+  FaCheckCircle as CheckIcon,
+  FaSpinner as Spinner,
+} from 'react-icons/fa'
+import { useMutation } from 'react-query'
+import { ToastHelper } from '~/components/helpers/toast.helper'
+import { useAuth } from '~/contexts/AuthenticationContext'
+import { useStorageService } from '~/utils/withStorageService'
+import { useWalletService } from '~/utils/withWalletService'
+
+interface DetailProps {
+  title: string
+  value: React.ReactNode
+}
+
+interface MetadataCardProps {
+  metadata: any
+  owner: string
+  network: number
+  isLockManager: boolean
+}
+
+const keysToIgnore = [
+  'token',
+  'lockName',
+  'expiration',
+  'keyholderAddress',
+  'lockAddress',
+  'checkedInAt',
+]
+
+const MetadataDetail = ({ title, value }: DetailProps) => {
+  return (
+    <div className="pb-2 border-b border-gray-400 last-of-type:border-none">
+      <span className="text-base">{`${title}: `}</span>
```

Comment:
Nit can be `{title}:` instead of templating use. use gap for spacing instead of blank space if required.